spesialist-felles/src/main/kotlin/no/nav/helse/HelseDao.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.helse
+
+import kotliquery.Row
+import kotliquery.queryOf
+import kotliquery.sessionOf
+import javax.sql.DataSource
+
+abstract class HelseDao(private val dataSource: DataSource) {
+
+    fun String.update(argMap: Map<String, Any?> = emptyMap()) = sessionOf(dataSource).use { session ->
```

Comment:
Kan vi gjøre disse protected? Så blør det ikke ut i andre klasser

Reply:
Jeg har egentlig ikke så lyst til dét, for det er så nyttig å kunne extende daoene i test-scope. Du skal uansett ikke se disse andre steder enn inne i DAOene.